docs-user/guide-remote-profiling.md

```diff
@@ -102,7 +102,19 @@ adb shell am start -n org.mozilla.geckoview_example/.App \
 
 ### Startup profiling Fenix
 
-Fenix has a [different way](https://firefox-source-docs.mozilla.org/mobile/android/geckoview/consumer/automation.html#reading-configuration-from-a-file) to specify environment variables: it uses a yaml file. The filename depends on the bundle ID of your Fenix app. The instructions below assume you want to profile the Fenix Nightly app, with the bundle ID `org.mozilla.fenix`.
+Fenix has a [different way](https://firefox-source-docs.mozilla.org/mobile/android/geckoview/consumer/automation.html#reading-configuration-from-a-file) to specify environment variables: it uses a yaml file.
+
+The easiest way to set up startup profiling is to run the `<fenix-repo>/tools/setup-startup-profiling.py` script. For example:
```

Comment:
Not everybody profiling fenix will have a checked out fenix repository. What about linking `<fenix-repo>` to https://github.com/mozilla-mobile/fenix?
Also, if this file works standalone, what do you think about suggesting to download it and providing a direct link ? https://raw.githubusercontent.com/mozilla-mobile/fenix/master/tools/setup-startup-profiling.py

Reply:
gently ping @mcomella :) I could do these changes myself but first I should know if these suggestions are sensible. Thanks!

Reply:
@julienw Those were good ideas – I updated my PR. Sorry for the delay!